oneview/resource_server_profile.go

```diff
@@ -2140,9 +2166,10 @@ func resourceServerProfileRead(d *schema.ResourceData, meta interface{}) error {
 		mpSettings := make([]map[string]interface{}, 0)
 		if len(serverProfile.ManagementProcessor.MpSettings) != 0 {
 			// initializing schema variables...
-			adminAcc := make([]map[string]interface{}, 0)
-			directory := make([]map[string]interface{}, 0)
-			keyManager := make([]map[string]interface{}, 0)
+			adminAcc := make([]map[string]interface{}, 1)
```

Comment:
any reason why did we changed this ?

Reply:
same question? not sure why those have been touched